### PR TITLE
Main homepage

### PR DESCRIPTION
**Pull request by santhoshpemmaka**
adding: all features to the social media app.

### CODE REVIEW

File: src/Components/HomePage/HomeLandingPage.scss [hunk 1]
```diff
@@ -0,0 +1,93 @@
+.home-container {
+	width: 100%;
+	height: 100%;
+	position: fixed;
+	background-color: #eff2f5;
+	.home-component {
+		max-width: 1024px;
+		margin: 0 auto;
```
Comment (Shubhamkr2610) [inline src/Components/HomePage/HomeLandingPage.scss:8]: you can use rem here 
or vw unit

File: src/index.js [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 import {makeServer} from "./server";
+
 makeServer();
 ReactDOM.render(
 	<React.StrictMode>
-		<Mock />
-		{/* <App /> */}
+		<Router>
+			{/* <Mock /> */}
+			<App />
```
Comment (Shubhamkr2610) [inline src/index.js:15]: remove unused code

**Review by Shubhamkr2610 — APPROVED** [review]
code looks good  to  me

**Review by Shubhamkr2610 — APPROVED** [review]

**Review by Shubhamkr2610 — APPROVED** [review]

File: src/Components/HomePage/HomeLandingPage.scss (hunk 1 above)
Comment (santhoshpemmaka) [reply src/Components/HomePage/HomeLandingPage.scss:8]: Okay bro

**Comment by VarshithSSV** [comment]
Code looks good and easy understand flow of the code.

Great work 💯 . Good to merge 👍

**Review by VarshithSSV — APPROVED** [review]

**Pull request merged**